imageroot/systemd/user/freepbx.service

```diff
@@ -33,6 +33,9 @@ ExecStart=/usr/bin/podman run \
     --volume=/dev/log:/dev/log \
     --volume=./notify:/notify:z \
     --volume=./certificates:/etc/asterisk/keys:z \
+    --volume=./report/api-config.json:/opt/nethvoice-report.json:z \
+    --volume=reports_config:/opt/nethvoice-report:z \
+    --volume=tancredi:/var/lib/tancredi:z \
```

Comment:
Why a new volume has been added for trancredi in this PR?

Reply:
That's because [queue-miner.php](https://github.com/nethesis/nethvoice-report/blob/ns8/root/opt/nethvoice-report/scripts/queue-miner.php) looks in both reports and tancredi config files :sweat_smile: